BuildForSDG/SmeVest-Backend
# chore: create required database models

## Description
- Created User, Message, Investor Models
- Created Project, Sme, Connection, Investment Models
- Added some methods for user authentication to User Model
- Optimized the base route unit test
- Wrote unit tests for the database models
- Added some rules to eslintrc.js configuration

Delivers #5

## How Has This Been Tested?
1. In your terminal, run `git clone https://github.com/BuildForSDG/SmeVest-Backend.git`
2. Run `cd SmeVest-Backend`
3. Run `git fetch origin ch-create-models`
4. Run `git checkout origin/ch-create-models`
5. Run `yarn install`
6. create a `.env` file in the root directory then add details as seen in `.env-example` file
6. Run `yarn run start:dev` to start the development server, you should get a response of **"Database connection successful"** on your terminal
7. Run `yarn run test` to run unit tests

## Screenshots (if applicable, else remove this line / section)
None

## Checklist:
